### PR TITLE
Threat Hunting!!!1

### PR DESCRIPTION
**Pull request by jacknagz**
### Background

A scenario for our upcoming webinar!

### Changes

* Send Data script
* Synthetic data
* Scenario

### Testing

* Tested IRL

### CODE REVIEW

File: test_scenarios/send_data.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        logging.debug('Message [%d] response: %s', indx + 1, resp['ResponseMetadata']['HTTPStatusCode'])
+
+def process_any_raw(event_time_shift, client, queue_url, logs, log_type):
+    logs = time_shift_raw_logs(event_time_shift, logs, log_type)
+    logging.info('Sending [%d] %s logs...', len(logs), log_type)
+    for indx, log in enumerate(logs):
+        resp = send_message(client, queue_url, log, 'raw')
+        logging.debug('Message [%d] response: %s', indx + 1, resp['ResponseMetadata']['HTTPStatusCode'])
+
+# FIXME: we need to change to S3 files, sqs does not guarantee order of events and VPC flow needs a header to precede
```
Comment (nhakmiller) [inline test_scenarios/send_data.py:87]: Could we use a FIFO SQS queue? Not sure if Panther supports this for SQS sources.

**Review by nhakmiller — APPROVED** [review]
Super cool!

File: test_scenarios/send_data.py (hunk 1 above)
Comment (jacknagz) [reply test_scenarios/send_data.py]: We'd have to create the queues as FIFO in our integration

**Review by rleighton — APPROVED** [review]

File: test_scenarios/send_data.py (hunk 1 above)
Comment (rleighton) [reply test_scenarios/send_data.py]: Switched to S3 from SQS.

**Pull request merged**